<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据前处理窗口</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sure_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确认分析</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reload_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重新加载</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>711</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>消息框</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>12</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选取文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>12</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="choosepath">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>37</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导出结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="data2db">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入数据库</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
